### DISCUSSION TITLE
Merging for mosaic

### DISCUSSION THREAD

**Discussion by Ixiodor** [corteva/rioxarray#239] [Q&A]
I'm tring to geospatially merge some dataset to have a bigger mosaic. I opened all the images, clipped them(saved them to check if all was fine, for merge I didn't re-opened the images) and tried to merge. But i get this error:
```
        with dataset_opener(datasets[0]) as first:
>           first_profile = first.profile
E           AttributeError: 'RasterioDatasetDuck' object has no attribute 'profile'
```
Why?

**Comment by snowman2** [comment]
This has been addressed in rioxarray version 0.2 (https://github.com/corteva/rioxarray/pull/217; https://github.com/corteva/rioxarray/issues/225).
